torch/csrc/jit/runtime/interpreter.cpp

```diff
@@ -208,6 +208,37 @@ struct InterpreterStateImpl : c10::intrusive_ptr_target {
     checkAndStartRecordFunction(frames.back(), stack);
   }
 
+#if defined(__GNUC__) || defined(__clang__)
+#define USE_COMPUTED_GOTO
+#endif
+// Primitives for making interpreter internal state transitions.
+// We maintain two local variables as the internal interpreter state:
+// `frame` will be the current frame that the interpreter operatos on.
+// `inst` will the current instruction pointed to by program counter.
+//
+// Instruction blocks should be always declared through `INST` macro and
+// the instruction body should always start with a `INST_GUARD` declaration.
+// Also blocks should be ended properly with either `INST_NEXT` (for going
+// to the next instruction), or `INST_DISPATCH` (for jumping to a computed
+// position using `INST_FETCH`).
+#define INST_FETCH(X) (frame.function->instructions_[frame.pc += (X)])
```

Comment:
Optional:
Maybe code will be cleaner if change `INST_FETCH(X)` to:

`inst = (frame.function->instructions_[frame.pc += (X)])`